packages/@okta/vuepress-site/docs/concepts/oauth-openid/index.md

```diff
@@ -4,218 +4,231 @@ title: OAuth 2.0 and OpenID Connect Overview
+Single-Page Applications (SPAs), mobile, and platform-specific applications are **public** applications where end users can view and possibly modify their code. Any secrets in their code are open to malicious users. By comparison, server-side (web) applications are **confidential** or **private** applications. Private application clients can use client-side authentication methods such as client secrets.
 
-#### Is your client a SPA or native?
+#### Does your client use the redirect or embedded authentication model?
 
-If your client application is a SPA or a native application, you should use an authorization flow with PKCE, such as either the [Interaction Code flow with PKCE](#interaction-code-flow) or the [Authorization Code flow with PKCE](#authorization-code-flow-with-pkce). If you are doing a redirect flow to an Okta-hosted sign-in page, the Authorization Code flow with PKCE is recommended. If you want to embed the sign-in experience, the Interaction Code flow is recommended.
+If your client application is a SPA or a platform-specfic application that redirects its authentication requests to an Okta-hosted sign-in page ([the redirect model](/docs/concepts/redirect-vs-embedded/#redirect-authentication)), use the [Authorization Code flow with PKCE](#authorization-code-flow-with-pkce). If your application hosts the authentication flow itself ([the embedded model](/docs/concepts/redirect-vs-embedded/#embedded-authentication)), use the [Interaction Code flow](#interaction-code-flow).
 
 #### Does the client have an end user?
 
-If your client application is running on a server with no direct end user, then it can be trusted to handle credentials and use them responsibly. If your client application is only doing machine-to-machine interaction, then you should use the [Client Credentials flow](#client-credentials-flow).
+If your client application runs on a server with no direct end user, then it can be trusted to handle credentials and use them responsibly. If your client application is only doing machine-to-machine interaction, then you should use the [Client Credentials flow](#client-credentials-flow).
 
 #### Is your app high-trust?
 
-If you own both the client application and the resource that it's accessing, then your application can be trusted to handle your end user's username and password. These types of apps are considered "high-trust". Because of the high degree of trust required, you should only use the [Resource Owner Password flow](#resource-owner-password-flow) if other flows aren't viable.
+Applications are **high-trust** if you own both them and the resource they access. Because you own them, you can trust them to handle your end users' usernames and passwords. In this case, and _only if other flows aren't viable_, you may use the [Resource Owner Password flow](#resource-owner-password-flow).
```

Comment:
```suggestion
Applications are **high-trust** if you own both them and the resource they access. Because you own them, you can trust them to handle your end users' usernames and passwords. In this case, and _only if other flows aren't viable_, you may use the [Resource Owner Password flow](#resource-owner-password-flow). However, it isn't possible to use this flow with multifactor authentication, so you will likely want to look at alternatives such as the [Authorization Code](/docs/guides/implement-grant-type/authcode/main/) or [Interaction Code flow](#interaction-code-flow).
```

Reply:
Updated